klock/src/commonMain/kotlin/com/soywiz/klock/internal/MicroStrReader.kt

```diff
@@ -21,6 +21,13 @@ internal class MicroStrReader(val str: String, var offset: Int = 0) {
 		readChar()
 		return true
 	}
```

Comment:
In the implementation of tryRead if the next char it's not equal to the char provided to the method returns false and doesn't read the char. For this case I used another function to read if the current char is 'd' and the next char is 'o'.